sorcery.gemspec

```diff
@@ -22,6 +22,7 @@ Gem::Specification.new do |s|
   s.description = 'Provides common authentication needs such as signing in/out, activating by email and resetting password.'
   s.summary = 'Magical authentication for Rails applications'
   s.homepage = 'https://github.com/Sorcery/sorcery'
+  s.metadata = {"bug_tracker_uri" => "https://github.com/Sorcery/sorcery/issues", "changelog_uri" => "https://github.com/Sorcery/sorcery/blob/master/CHANGELOG.md"} if s.respond_to? :metadata=
```

Comment:
Since [metadata has been supported since 2011](https://github.com/rubygems/rubygems/commit/be483b99d8a53bce3e6faf9e6f3c3a5df452974e), I believe `if s.respond_to? ...` is unnecessary.